vercel.json

```diff
@@ -1323,6 +1323,11 @@
             "destination": "/features/featuresDeepDive/mesh/trailMesh",
             "statusCode": 301
         },
+        {
```

Comment:
@RaananW Do we have to update this `vercel.json` file? I never did it myself (I don't know what this file is for)...

Reply:
Same question at @Popov72 :)

Reply:
No need if you don't need the redirect. Changing the vercel is mainly to support the older URLs and staying "back-compat" in our URL scheme. Not needed here.